benchflow-dsl/src/main/scala/cloud/benchflow/dsl/ExplorationSpace.scala

```diff
@@ -100,15 +100,14 @@ object ExplorationSpace {
   }
 
   /**
-   * Generates and experiment bundle from the given exploration space point. Assumes the point exists in the
-   * exploraiton space.
+   * Generates and experiment bundle from the given exploration space point.
    *
    * @param explorationSpace         the exploration space where the experiment is
    * @param explorationSpacePoint    the point in the exploration space from which to generate the experiment
    * @param testDefinitionYamlString the test definition to build the experiment definition from as a YAML string
    * @param dockerComposeYamlString  the docker compose to build the experiment docker-compose from as a YAML string
    * @return a tuple of experiment definition YAML string, docker compose YAML string and experiment number
-   *         (index in exploration space) as an Option. None if point could not be found.
+   *         (index in exploration space) as an Option. None if point could not be found, or if multiple were found.
```

Comment:
The important thing to me is that in the caller you can distinguish the two cases, because you need for sure to generate different error messages.